pkgs/misc/tmux-plugins/default.nix

```diff
@@ -44,14 +44,12 @@ let
       dependencies = [ pkgs.bash ] ++ dependencies;
     }));
 
-  buildTmuxPluginFrom2Nix = a: buildTmuxPlugin ({
-    buildPhase = ":";
-    configurePhase =":";
-  } // a);
 
 in rec {
 
-  battery = buildTmuxPluginFrom2Nix {
+  mkDerivation = a: (buildTmuxPlugin a);
```

Comment:
This is exactly the same as `mkDerivation = buildTmuxPlugin` because of currying. This means you can just declare `mkDerivation` above directly, then `inherit mkDerivation;` here. Or declaring `mkDerivation` here would work as well (because it's in a `rec`).